### PR TITLE
After run finished, send finish to new end end point

### PR DESCRIPTION
**Pull request by odscjames**
https://github.com/open-contracting/kingfisher-scrape/issues/57

### CODE REVIEW

File: kingfisher_scrapy/base_spider.py [hunk 1]
```diff
@@ -1,6 +1,45 @@
...
+        response = requests.post(
+            called_on_class.crawler.settings['KINGFISHER_API_URI'] + '/api/v1/submit/end_collection_store/',
+            data=data,
+            headers=headers)
+
+        if not response.ok:
+            spider.logger.warning(
+                "Failed to post End Collection Store. API status code: {}".format(response.status_code))
+
+# Now we have our own base spiders (that use our generic functions)
```
Comment (BibianaC) [inline kingfisher_scrapy/base_spider.py]: I would delete this sentence.

File: kingfisher_scrapy/base_spider.py [hunk 2]
```diff
@@ -1,6 +1,45 @@
 import scrapy
 import requests
 
+# This file contain base spiders all our spiders extend from, so we can add some custom functionality
+#
+# However, we need to do this for several base spiders. To avoid copying and pasting code between base spiders,
```
Comment (BibianaC) [inline kingfisher_scrapy/base_spider.py]: I would delete this sentence. The first line is enough to explain what is in the file.

**Comment by odscjames** [comment]
If we delete those lines, there is nothing to explain why the generic functions are there. Someone could probably guess from looking at both classes, but I think it's good to comment to explain. And comments are usually good - we are trying to get more documentation at the moment!

**Comment by BibianaC** [comment]
I'm all about comments but comments are good when they are needed. In this case is self explanatory.

**Pull request merged**